src/org/loklak/api/search/GenericScraper.java

```diff
@@ -48,6 +48,8 @@
 
 public class GenericScraper extends HttpServlet {
 
+	private static final long serialVersionUID = 4653635987712691127L;
```

Comment:
@sudheesh001 The issue suggests that all the URLs should be crawled, so why keep them?